package Day45.question01;

public class Person extends Exception {
    void front_1(String input_front, String input_front2) {System.out.println("왜 안돼?");}
    void backend_1(String input_back, String input_back2) {System.out.println("왜 안돼?");}
    void database_1(String input_db) {System.out.println("왜 안돼?");}
    void manager_1(String input_mg) {System.out.println("왜 안돼?");}
}
